templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{%  extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>

```

Comment:
Indentation issue: According to the checklist, you should use 2 spaces for indentation in your `.html` files. Please adjust the indentation accordingly.